tests/ensemble_evaluator/conftest.py

```diff
@@ -184,27 +183,6 @@ def _dump_ext_job(ext_job, index):
     }
 
 
-def _mock_ws(host, port, messages, delay_startup=0):
```

Comment:
It would be perfectly valid to move it, but you could also do:
```
@pytest.fixture()
def _mock_ws():
    def wrapper(host, port, messages, delay_startup=0):
```
and make it a fixture inside `conftest`

Reply:
Is this true for all or only mock_ws?

Reply:
It is true of all of them, in that all could be converted to this pattern, but some might be more suited as `fixtues` than others, `mock_ws` struck me as very well suited, as you are just passing on a function. Not a very good answer, but I think neither approach is wrong.

Reply:
I am keeping the changes as it currently stands, based on the notion that this implementation even tho small is creeping outside of the scope of this issue. This issue is to remove the loading of tests module together with ert when installing `pip install -e .` ert